tests/Agent/IntegrationTests/ContainerIntegrationTests/LinuxKafkaTests.cs

```diff
@@ -0,0 +1,128 @@
+// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using NewRelic.Agent.ContainerIntegrationTests.ContainerFixtures;
+using NewRelic.Agent.IntegrationTestHelpers;
+using NewRelic.Testing.Assertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace ContainerIntegrationTests;
+
+public abstract class LinuxKafkaTest<T> : NewRelicIntegrationTest<T> where T : LinuxKafkaTestFixtureBase
+{
+    private const int TopicNameLength = 15;
+
+    internal string _topicName;
+    private readonly T _fixture;
+    private readonly static Random _rnd = new();
+
+    protected LinuxKafkaTest(T fixture, ITestOutputHelper output) : base(fixture)
+    {
+        _fixture = fixture;
+        _fixture.TestLogger = output;
+
+        _topicName = GenerateTopic();
+        var brokerName = "broker" + _topicName;
+        ((ContainerApplication)_fixture.RemoteApplication).DockerDependencies.Add(brokerName);
+
+        _fixture.Actions(setupConfiguration: () =>
+            {
+                var configModifier = new NewRelicConfigModifier(_fixture.DestinationNewRelicConfigFilePath);
+                configModifier.SetLogLevel("debug");
+                configModifier.ConfigureFasterMetricsHarvestCycle(10);
+                configModifier.LogToConsole();
```

Comment:
Was `LogToConsole` for debugging or does it need to be on for the test?

Reply:
I believe it is required.  The smoke tests also use it.